src/styles/blocks/card.scss

```
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 19px;
+  font-size: $font-size;
+  line-height: $line-height;
+  color: $main-color;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__header {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    color: $secondary-color;
+  }
+
+  &__reviews-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    margin: 0;
+  }
+
+  &__price-container{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__button {
+    display: block;
+    margin: 0 auto;
+    width: 166px;
```

full available width
```suggestion
    width: 100%;
```

---

probably you've not seen this comment . Please add a commit to fix it